crates/fluvio-cli/src/connector/mod.rs

```diff
@@ -94,8 +94,6 @@ pub struct ConnectorConfig {
     pub(crate) topic: String,
     pub(crate) version: Option<String>,
     #[serde(default)]
-    pub(crate) create_topic: bool,
```

Comment:
Serde just ignores items when they're in the thing still, right?

Reply:
yep - it's non-exhaustive like structopt is.

If we want to enforce unknown fields deny then we need the below:

```
#[serde(deny_unknown_fields)]

Always error during deserialization when encountering unknown fields. When this attribute is not present, by default unknown fields are ignored for self-describing formats like JSON.

Note: this attribute is not supported in combination with [flatten](https://serde.rs/field-attrs.html#flatten), neither on the outer struct nor on the flattened field.
```